subt/submission/finals.toml

```diff
@@ -8,7 +8,7 @@ B60W1C2400LH3600W = "freyja"
 A0W1C2400RH3600W = "freyja"
 C300W1C1500RH3600W = "freyja"
 E600W900LH3600W = "pam"
-H1860W600RH3600W = "pam"
-G1800W600LH3600W = "pam"
+H1980W600RH3600W = "pam"
+G1920W600LH3600W = "pam"
```

Comment:
The difference between starting times of the second first-wave drone (F) and first second-wave drone (G): (1920 - 660) / 60 = 21 minutes. This looks as intended.